### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by FireInstall**
remove smoke particles spawning every time a sign is edited 
fix signs not getting waxed on manuel creating them
fix double doors being too loud
better logging of exceptions

sorry Kopfi

### CODE REVIEW

**Comment by FireInstall** [comment]
[Lockette fixes](https://trello.com/c/4TPSLjy1/245-lockette-fixes)

File: src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java [hunk 1]
```diff
@@ -227,23 +232,37 @@ public void onAttemptBreakSign(BlockBreakEvent event) {
...
+    public void onAttemptChangeLockerSign(SignChangeEvent event) {
         Block block = event.getBlock();
-        if(LocketteProAPI.isLockSign(block)||LocketteProAPI.isAdditionalSign(block)){
-            Sign sign = (Sign)block.getState();
+        if (LocketteProAPI.isLockSign(block) || LocketteProAPI.isAdditionalSign(block)) {
+            Sign sign = (Sign) block.getState();
             sign.setWaxed(true);
             sign.update();
             event.setCancelled(true);
+
```
Comment (joshi1999) [inline src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java:243]: Then just remove it ^^

File: src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java [hunk 2]
```diff
@@ -227,23 +232,37 @@ public void onAttemptBreakSign(BlockBreakEvent event) {
...
+    public void onAttemptBreakWaxedLockerSign(PlayerInteractEvent event) {
         Action action = event.getAction();
         Block block = event.getClickedBlock();
-        boolean isAxe = Objects.equals(event.getItem(), new ItemStack(Material.WOODEN_AXE)) || Objects.equals(event.getItem(), new ItemStack(Material.STONE_AXE)) || Objects.equals(event.getItem(), new ItemStack(Material.IRON_AXE)) || Objects.equals(event.getItem(), new ItemStack(Material.GOLDEN_AXE)) || Objects.equals(event.getItem(), new ItemStack(Material.DIAMOND_AXE)) || Objects.equals(event.getItem(), new ItemStack(Material.NETHERITE_AXE));
-        if(action == Action.RIGHT_CLICK_BLOCK&&(LocketteProAPI.isLockSign(block)||LocketteProAPI.isAdditionalSign(block))&&isAxe){
+
+        boolean isAxe = false;
+        if (event.getItem() != null) {
+            switch (event.getItem().getType()) {
+                case WOODEN_AXE, STONE_AXE, IRON_AXE, GOLDEN_AXE, DIAMOND_AXE, NETHERITE_AXE -> isAxe = true;
```
Comment (joshi1999) [inline src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java]: You could use the tag: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Tag.html#ITEMS_AXES

File: src/main/java/me/crafter/mc/lockettepro/LocketteProAPI.java [hunk 3]
```diff
@@ -462,7 +464,7 @@ public static void toggleDoor(Block block, boolean open) {
         org.bukkit.block.data.Openable openablestate = (org.bukkit.block.data.Openable) block.getBlockData();
         openablestate.setOpen(open);
         block.setBlockData(openablestate);
-        block.getWorld().playSound(block.getLocation(), open ? Sound.BLOCK_WOODEN_DOOR_OPEN : Sound.BLOCK_WOODEN_DOOR_CLOSE, 100,1);
+        block.getWorld().playSound(block.getLocation(), open ? Sound.BLOCK_WOODEN_DOOR_OPEN : Sound.BLOCK_WOODEN_DOOR_CLOSE, 1, 1);
```
Comment (joshi1999) [inline src/main/java/me/crafter/mc/lockettepro/LocketteProAPI.java:467]: 😱

**Review by joshi1999 — CHANGES_REQUESTED** [review]

File: src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java (hunk 1 above)
Comment (FireInstall) [reply src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java:243]: I left it as a comment in, so in future merges it doesn't crawl back out of it's grave

File: src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java (hunk 2 above)
Comment (FireInstall) [reply src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java]: Nice one. I was wondering if something like this existed!

Comment (joshi1999) [reply src/main/java/me/crafter/mc/lockettepro/BlockPlayerListener.java:259]: Ah, I see ^^

File: src/main/java/me/crafter/mc/lockettepro/LocketteProAPI.java (hunk 3 above)
Comment (FireInstall) [reply src/main/java/me/crafter/mc/lockettepro/LocketteProAPI.java:467]: \*[flashy-thing](http://2.bp.blogspot.com/-pWmrlshMj9M/UBlIQRZYlkI/AAAAAAAAAI8/8Qm9abuzmfw/s1600/will-smith-men-in-black-3-flash.jpg)s you\*
You saw nothing. You will live a happy life and fix the Minigames-Plugin.

**Review by joshi1999 — APPROVED** [review]

**Pull request merged**